test/ibmq/test_ibmq_backend_validities.py

```diff
@@ -14,59 +14,52 @@
 
 """IBMQ Remote Backend Qobj Tests."""
 
-import os
-
 from qiskit import (BasicAer, ClassicalRegister, QuantumCircuit,
                     QuantumRegister)
 from qiskit.providers.ibmq import IBMQ
-from qiskit.qasm import pi
 from qiskit.test import QiskitTestCase, slow_test
-from qiskit.compiler import transpile
+from qiskit.compiler import assemble, transpile
 
 from ..decorators import requires_qe_access
 
 
-class TestIBMQQobj(QiskitTestCase):
+class TestIBMQBackendValidities(QiskitTestCase):
     """Qiskit backend qobj test.
 
-    Compares remote simulator as configured in environment variables
-    'IBMQ_QOBJ_DEVICE', 'IBMQ_TOKEN' and 'IBMQ_QOBJ_URL' against local
-    simulator 'local_qasm_simulator' as ground truth.
+    Executes a series of circuits of special interest using remote
+    simulators as configured in environment variables
+    'IBMQ_QOBJ_DEVICE', 'IBMQ_TOKEN' and 'IBMQ_QOBJ_URL', comparing
+    results against local simulator 'local_qasm_simulator' as ground
+    truth.
```

Comment:
Can you revise this docstring? The environment variables are not used anymore, and instead we just use the regular credentials mechanism in the same way as the rest of the tests - and also the heading `Qiskit backend qobj test` is a bit misleading.